src/error.rs

```diff
@@ -23,28 +23,26 @@ pub enum Error {
 }
 
 impl Error {
+    #[deprecated(since = "0.0.22", note = "Please use the to_string() method instead")]
```

Comment:
Sorry, I didn't look into this properly last time. Just delete this method. The `Error` trait provides a default implementation.

Reply:
I think this would break compatibility if other dependent crates depends on it.
But more breaking change would be renaming `dyn` module.

Reply:
You're right, and if it were version 1.0 then I would agree with keeping it. But it's version 0.0.22, and I think it's unlikely anyone is using this instead of `Display`.

Reply:
Let me look at this, philipc might be right here

Reply:
Done.